scripts/gdb_printers.py

```diff
@@ -59,5 +61,259 @@ def build_pretty_printer():
     pp.add_printer('TurnLaneData', '::TurnLaneData$', TurnLaneDataPrinter)
     return pp
 
-#gdb.pretty_printers = [filter(lambda x: x.name != 'OSRM', gdb.pretty_printers)]
```

Comment:
I don't really understand the purpose of this line, so I am not sure why you removed it. Could you give some context?